cdios.egg-info/PKG-INFO

```diff
@@ -0,0 +1,12 @@
+Metadata-Version: 1.1
```

Comment:
Whoops, I'm going to add `*.egg-info/` to the .gitignore so we don't check these files in. Will comment once I do that, then once you merge master these changes should disappear from the PR.

Reply:
I removed the egg-info/ directory in my latest commit so hopefully it does not show up any further.